static/markdown/tap-target-maintenance.md

```diff
@@ -3,38 +3,141 @@ title: "Maintaining Taps & Targets"
+- The `catalog` input file was originally called `properties` so some will accept either
+- [Getting started guide](https://github.com/singer-io/getting-started)
+- [tap template](https://github.com/singer-io/singer-tap-template)
+
+#### [Meltano SDK](https://sdk.meltano.com/en/latest/) Based Connectors
+
+A software development kit that abstracts many of the challenges of building Singer connectors so the developer doesn't need to know about them and also provides accelerators to speed up the process of writing a new connector. See the [Meltano SDK](https://sdk.meltano.com/en/latest/) docs for more details. Example of some features:
+
+- cookiecutter templates to seed a connector repo
+- manages translating records into Singer spec compliant messages
+- methods for common API functionality (i.e. parent child relationships, pagination, retry, post processing, etc.)
+- methods for common SQL functionality
+- default unit tests
+- additional features for testing, printing config metadata,
+
+#### [Pipelinewise](https://github.com/transferwise/pipelinewise)
+
+- An open source tool created in 2019 that helps configure and run Singer connectors, similar to Meltano
+- Maintains its own list of connectors that it supports. You can also run all of these with Meltano!
+- Well documented patterns around [metadata columns](https://transferwise.github.io/pipelinewise/user_guide/metadata_columns.html), [replication methods](https://transferwise.github.io/pipelinewise/concept/replication_methods.html), and [schema changes](https://transferwise.github.io/pipelinewise/user_guide/schema_changes).
+
+## Evaluating a tap or target
+
+These are things you can look for when evaluating a tap or target.
+Many of these recommendations can be used for evaluating open source projects in general.
+
+You should start by looking on MeltanoHub to find and evaluate connectors.
+Although a connector can still be listed if its not well maintained so click through to the repo to inspect it closer if you're unsure.
+None of these attributes alone will tell you what the best connectors are, so look at many of them to make a holistic evaluation.
+
+- <u>Organization backed connectors</u>: If its in the namespace of `meltano`/`meltanolabs`/`transferwise`/`singer-io`/ etc. its usually a good bet that the connector is maintained because one of the organizations is invested in it's success. Although this isn't always the case, especially with some of the older singer-io variants.
+- <u>MeltanoHub default</u>: The Meltano community does it's best to label the most active variant as the default on MeltanoHub. Although it's still possible that the best option is unmaintained.
+- <u>Meltano SDK</u>: If it uses the Meltano SDK then its a good sign that its relatively recent and has good support for common features. If there are many TODOs still in the repo that came with the cookiecutter it might indicate that the connector was never finished or is a hobby project.
```

Comment:
```suggestion
- <u>Meltano Singer SDK</u>: If it uses the Meltano Singer SDK then its a good sign that its relatively recent and has good support for common features. If there are many TODOs still in the repo that came with the cookiecutter it might indicate that the connector was never finished or is a hobby project.
```